allennlp/models/semantic_parsing/wikitables/wikitables_erm_semantic_parser.py

```diff
@@ -127,6 +128,10 @@ def __init__(self,
                                                                dropout=dropout)
         self._checklist_cost_weight = checklist_cost_weight
         self._agenda_coverage = Average()
+        # We don't need a separate beam search since the trainer does that already. But we're defining one just to
+        # be able to use interactive beam search (a functionality that's only implemented in the ``BeamSearch``
+        # class) in the demo. We'll use this only at test time.
+        self._beam_search = BeamSearch(beam_size=decoder_beam_size)  # type: ignore
```

Comment:
Why do you need `type: ignore` here?

Reply:
Mypy complained that it needs type annotation for that variable. I assumed that's a mypy bug since the variable is actually being defined there.

Reply:
Maybe try `self._beam_search: BeamSearch = ...`?

Reply:
Looks like that works. Thanks!